src/lib/server/events/data/sample-events.ts

```diff
@@ -1 +1,320 @@
\ No newline at end of file
+export const SAMPLE_EVENTS = [
```

Comment:
We for our sample events used in debug mode, we could move the below data to a new `src/lib/server/events/data/sample-events.json` and import it here for type-checking, to make sure our `SAMPLE_EVENTS_JSON` is up to date with type `ClubEvent`.
```suggestion
import type { ClubEvent } from '$lib/public/events/event';
import SAMPLE_EVENTS_JSON from './sample-events.json';

export const SAMPLE_EVENTS: ClubEvent[] = [...SAMPLE_EVENTS_JSON];
```